packages/ui-extensions/src/surfaces/point-of-sale/event/input/PurchaseCompleteInput.ts

```diff
@@ -0,0 +1,11 @@
+import {BaseInput} from './BaseInput';
+
+export interface PurchaseCompleteInput extends BaseInput {
+  lineItems: any[];
```

Comment:
I think the way this extends BaseInput means that all these fields will be at the root of the input which might be awkward. ie you would see something like `input.session.shopId` and `input.lineItems` which feels strange. Should we instead put all these fields in an object like this:

```
cart: {
  lineItems: any[],
  discounts: any[];
  attributedStaff: any;
  orderId: any;
  draftCheckoutId: any;
  taxLines: any[];
  shippingLines: any[];
}
```